akka-actor/src/main/scala/akka/actor/Props.scala

```diff
@@ -156,10 +112,9 @@ object Props {
  * }}}
  */
 @SerialVersionUID(2L)
-final case class Props(deploy: Deploy, clazz: Class[_], args: immutable.Seq[Any]) {
+final case class Props(deploy: Deploy = Props.defaultDeploy, clazz: Class[_], args: immutable.Seq[Any]) {
 
-  if (Modifier.isAbstract(clazz.getModifiers))
-    throw new IllegalArgumentException(s"Actor class [${clazz.getName}] must not be abstract")
+  Props.isAbstract(clazz)
```

Comment:
```validate``` is ok to me!

I had to make some really tiny modifications to keep everything working properly, this is not strictly needed, but this is spare code in the body of a class, refactoring it out to a companion object method could be considered good practice, moreover it is in the AbstractProps because is a really JVM dependant check.
Does this sounds good to you?